client/src/AppBundle/Resources/views/Admin/Index/editUser.html.twig

```diff
@@ -22,6 +23,11 @@
 
     {{ form_input(form.firstname,'addUserForm.firstname') }}
     {{ form_input(form.lastname,'addUserForm.lastname') }}
+
+    {% if form.email is defined and app.user.roleName == class_const(UserClass, 'ROLE_SUPER_ADMIN') %}
```

Comment:
Is the role check necessary? In other forms I've just used `if form.email is defined` and let the form handle permissions exclusively

Reply:
No not especially, happy to remove it.